app/views/directives/notifications/notification-body.html

```diff
@@ -0,0 +1,61 @@
+<div
+  uib-dropdown
+  class="dropdown pull-right dropdown-kebab-pf"
+  ng-if="notification.actions && notification.actions.length > 0">
+  <button
+    uib-dropdown-toggle
+    class="btn btn-link dropdown-toggle"
+    type="button"
+    id="dropdownKebabRight"
+    data-toggle="dropdown"
+    aria-haspopup="true"
+    aria-expanded="true">
+    <span class="fa fa-ellipsis-v"></span>
+  </button>
+  <ul
+    class="dropdown-menu dropdown-menu-right"
+    aria-labelledby="dropdownKebabRight">
+    <li
+      ng-repeat="action in notification.actions"
+      role="{{action.isSeparator === true ? 'separator' : 'menuitem'}}"
+      ng-class="{'divider': action.isSeparator === true, 'disabled': action.isDisabled === true}">
+      <a
+        ng-if="action.isSeparator !== true"
+        class="secondary-action"
+        title="{{action.title}}"
+        ng-click="$ctrl.customScope.handleAction(notification, action)">
+        {{action.name}}
+      </a>
+    </li>
+  </ul>
+</div>
+<span
+  ng-if="notification.status"
+  class="{{'pull-left ' + $ctrl.customScope.getNotficationStatusIconClass(notification)}}"
+  ng-click="$ctrl.customScope.markRead(notification)"></span>
+<div
+  class="drawer-pf-notification-content"
+  ng-click="$ctrl.customScope.markRead(notification)">
+  <span
+    class="drawer-pf-notification-message"
+    tooltip-append-to-body="true"
+    tooltip-popup-delay="500"
+    tooltip-placement="bottom"
+    tooltip="{{notification.message}}">
+    <span>
+       {{notification.reason | sentenceCase}} for {{notification.kind}}
+     </span>
+     <a ng-href="{{notification.name | navigateResourceURL : notification.kind : notification.namespace}}">
+       {{notification.name}}
+     </a>
+  </span>
+  <span ng-if="$ctrl.drawerExpanded" class="drawer-pf-notification-message text-muted small">
+    {{notification.message}}
+  </span>
+  <div
+    class="drawer-pf-notification-info"
+    ng-click="$ctrl.customScope.markRead(notification)">
+    <span class="date">{{notification.lastTimestamp | date:'MM/dd/yyyy'}}</span>
+    <span class="time">{{notification.lastTimestamp | date:'h:mm:ss a'}}</span>
```

Comment:
```html
<span class="date">{{notification.lastTimestamp | date:'shortDate'}}</span>
<span class="time">{{notification.lastTimestamp | date:'mediumTime'}}</span>
```

Better to use these so format is properly localized.

https://docs.angularjs.org/api/ng/filter/date